packages/config/src/configDefaults.ts

```diff
@@ -33,11 +33,13 @@ export const getInitialConfig = ({
     resolver: null,
     artifactor: null,
     ethpm: {
-      ipfs_host: "ipfs.infura.io",
-      ipfs_protocol: "https",
-      registry: "0x8011df4830b4f696cd81393997e5371b93338878",
-      install_provider_uri:
-        "https://ropsten.infura.io/v3/26e88e46be924823983710becd929f36"
+      ipfsHost: "ipfs.infura.io",
+      ipfsProtocol: "https",
+      ipfsPort: "5001",
+      registry: {
+        address: "0x8011df4830b4f696cd81393997e5371b93338878",
```

Comment:
I'm happy to deploy an un-authenticated registry for truffle to use as its default, unless you guys want to handle that. I'm 50/50 on whether it should be on mainnet (better reliability / authority / helps minimize spam packages) or on any testnet (much much cheaper / easier to start getting your hands dirty with publishing ethpm packages)

Reply:
does this field respect ENS names?

Reply:
Yep, though right now it only permits ens on mainnet. However, upon further investigation it looks like truffle supports ens on all testnets except kovan, so I'll make the update to suppport ens on these other testnets.